### PR TITLE
[2201.8.x] Update const BBE

### PR DESCRIPTION
**Pull request by prakanth97**
## Purpose
Add list constant and map constant examples

### CODE REVIEW

File: examples/const-and-final/const_and_final.bal [hunk 1]
```diff
@@ -4,12 +4,18 @@ import ballerina/io;
 const MAX_VALUE = 1000;
```
Comment (MaryamZi) [inline examples/const-and-final/const_and_final.bal:4]: Not introduced by this PR, but do we know why both don't have the type here? I think one can have the type.

File: examples/const-and-final/const_and_final.bal [hunk 2]
```diff
@@ -4,12 +4,18 @@ import ballerina/io;
...
 const URL = "https://ballerina.io";
 
+// Map and list can be declared as constant.
```
Comment (MaryamZi) [inline examples/const-and-final/const_and_final.bal:7]:
```suggestion
// Mapping and list constants can also be defined.
```

File: examples/const-and-final/const_and_final.bal (hunk 1 above)
Comment (prakanth97) [reply examples/const-and-final/const_and_final.bal:4]: +1

Comment (prakanth97) [reply examples/const-and-final/const_and_final.bal]: Updated [d250c9f](https://github.com/ballerina-platform/ballerina-distribution/pull/4713/commits/d250c9f661229b921af0f6dc9bea2f0b491595fb)

**Review by hasithaa — APPROVED** [review]

**Comment by rdulmina** [comment]
Closing this PR since these changes are sent to the master branch and we are recreating `2201.8.x` from that master later.

**Pull request closed without merging**